proto/regen/group/v1alpha1/query.proto

```diff
@@ -0,0 +1,244 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+import "regen/group/v1alpha1/types.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+// Query is the regen.group.v1alpha1 Query service.
+service Query {
+
+  // GroupInfo queries group info based on group id.
+  rpc GroupInfo(QueryGroupInfoRequest) returns (QueryGroupInfoResponse);
+
+  // GroupAccountInfo queries group account info based on group account address.
+  rpc GroupAccountInfo(QueryGroupAccountInfoRequest) returns (QueryGroupAccountInfoResponse);
+  
+  // GroupMembers queries members of a group
+  rpc GroupMembers(QueryGroupMembersRequest) returns (QueryGroupMembersResponse);
+
+  // GroupsByAdmin queries groups by admin address.
+  rpc GroupsByAdmin(QueryGroupsByAdminRequest) returns (QueryGroupsByAdminResponse);
```

Comment:
It seams we are trying to implement relational DB - this queries should be handled by an off-chain indexer, or only added later when needed for state machine operations.

Reply:
That's a bigger question. This decision was made a long time ago